static/src/js/components/Map/ZoomHome.js

```diff
@@ -127,15 +122,13 @@ class ZoomExtended extends Control.Zoom {
 
     // return this._map.setView([-90, 0], 0)
     // }
+
+    // TODO This isn't great for different projections, existing problem
```

Comment:
Is there a ticket listed?

Reply:
Don't have a ticket listed, but I'll take a look at actually fixing the TODO above, which would render this obsolete and be the better fix